packages/preset-mini/src/_variants/data.ts

```diff
@@ -25,11 +25,11 @@ function taggedData(tagName: string): Variant {
     match(matcher, ctx: VariantContext<Theme>) {
       const variant = variantGetParameter(`${tagName}-data-`, matcher, ctx.generator.config.separators)
       if (variant) {
-        const [match, rest] = variant
+        const [match, rest, label] = variant
         const dataAttribute = h.bracket(match) ?? ctx.theme.data?.[match] ?? ''
```

Comment:
<img width="585" alt="image" src="https://github.com/user-attachments/assets/7dada812-8a92-4466-905d-268d3c1853d4">

I think this should be consistent with tailwind and the attributes should be wrapped in quotation marks.

Reply:
sure